<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/rec/Picture/list.png</normaloff>:/rec/Picture/list.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #b2b2b2;
border-radius:20px;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="goToTaskWriteWindow">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Добавити в список</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goToChangeListWindow">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>370</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff; /* цвет текста */
  text-decoration: none; /* убирать подчёркивание у ссылок */
  user-select: none; /* убирать выделение текста */
  background: rgb(212,75,56); /* фон кнопки */
  outline: none; /* убирать контур в Mozilla */
font-weight:900;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>Перейти до списку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goToQuickTaskWindow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff; /* цвет текста */
  text-decoration: none; /* убирать подчёркивание у ссылок */
  user-select: none; /* убирать выделение текста */
  background: rgb(212,75,56); /* фон кнопки */
  outline: none; /* убирать контур в Mozilla */
font-weight:900;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>Швидкі задачі</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-10</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:70px;
font-weight:900;</string>
    </property>
    <property name="text">
     <string>To Do List</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_quickTask">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_WriteTask">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>120</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="compTask">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
